packages/teleport-plugin-vue-base-component/src/utils.ts

```diff
@@ -145,19 +145,30 @@ export const generateConditionalNode = (
 ) => {
   const { reference, value } = node.content
   const conditionalKey = reference.content.id
+  let conditionalStatement: string = createConditionalStatement(
+    conditionalKey,
+    generateConditionalExpression(value, node.content.condition)
+  )
 
-  // 'v-if' needs to be added on a tag, so in case of a text node we wrap it with
-  // a 'span' which is the less intrusive of all
+  Object.values(node).forEach((child) => {
+    if (typeof child === 'object') {
```

Comment:
also, you can assume node.content.node is the next UIDLNode inside, since the input is of type UIDLConditionalNode.